src/main/kotlin/ai/digital/integration/server/deploy/internals/cluster/DeployDockerClusterHelper.kt

```diff
@@ -192,9 +193,22 @@ open class DeployDockerClusterHelper(val project: Project) : DockerClusterHelper
 
     private fun createNetwork() {
         if (!networkExists()) {
-            project.exec {
-                executable = "docker"
-                args = listOf("network", "create", ClusterConstants.NETWORK_NAME)
+            // Use ProcessBuilder instead of deprecated execOperations
+            val command = listOf("docker", "network", "create", ClusterConstants.NETWORK_NAME)
```

Comment:
Why isn't the **execOperations.exec** method being used in this case?

Reply:
Updated with execOperations.exec method